Makefile

```diff
@@ -0,0 +1,8 @@
+DEEPSPEECH_REPO ?= https://github.com/mozilla/DeepSpeech.git
```

Comment:
Should we move these docker related files to a "docker" subfolder instead of leaving them in the root?

Reply:
I'm not sure, is this really adding noise right now?

Reply:
It's not adding noise, just thinking that now we have a top level Makefile that only does a very specific thing.